modules/contact/test/tests/bouncing-block-contact/ping-ponging/kinematic-ping-pong.i

```diff
@@ -0,0 +1,109 @@
+starting_point = 2e-1
+offset = 1e-2
+
+[GlobalParams]
+  displacements = 'disp_x disp_y'
+  diffusivity = 1e0
+[]
+
+[Mesh]
+  file = long-bottom-block-no-lower-d.e
+[]
+
+[Variables]
+  [./disp_x]
+  [../]
+  [./disp_y]
+  [../]
+[]
+
+[ICs]
+  [./disp_y]
+    block = 2
+    variable = disp_y
+    value = ${fparse starting_point + offset}
+    type = ConstantIC
+  [../]
+[]
+
+[Kernels]
+  [./disp_x]
+    type = MatDiffusion
```

Comment:
I'm fine with this as a temporary solution, but can't we just make the `contact` module depend on `tensor_mechanics` so that these can be real problems? We could move all of our contact test cases from the `combined` module if we did that. Thoughts on this, @permcody? We're already doing that with the `xfem` module for similar reasons. That would also allow us to get a meaningful code coverage report from the `contact` module.

Reply:
We could move all of the combined/test/tests/mortar_tm tests here, and the subsequent tests I'll add shortly for frictional, RANFS, kinematic, etc.

While you're at it, should the thermal contact actions be moved to contact, and have contact depend on heat_conduction?

Reply:
> That would also allow us to get a meaningful code coverage report from the `contact` module.

This would provide added coverage for the old contact formulations/models. Mortar and RANFS objects are already covered in the current `contact` module. I'm supportive of creating the `tensor_mechanics` dependency...but I don't want to do that in this PR.

Reply:
I wouldn't advocate for mixing mechanical and thermal contact in the modules.  That would make it so an app that brings in the 'contact' module would also get heat_conduction, which they may not want.  I can't imagine an app bringing in 'contact' and not wanting 'tensor_mechanics', though.

Reply:
I could see the argument for just moving all of the contact code into 'tensor_mechanics', though.  Then it would be consistent with how 'heat_conduction' is structured.

Reply:
I just merged a PR in today that makes contact depend on tensor_mechanics.  It would make me feel so much better if these all used tensor_mechanics.